source/installation/index.html

```
@@ -13,7 +13,76 @@
+  
+  <div class="compare-installations">
+    <table>
+      <tr>
+        <th></th>
+        <th>HA OS<sup>1</sup></th>
+        <th>Container<sup>1</sup></th>
+      </tr>
+        <td><a href="/docs/automation" target="_blank">Automations</a></td>
+        <td>{% icon "openmoji:check-mark" %}</td>
+        <td>{% icon "openmoji:check-mark" %}</td>
+      </tr>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </tr> ]

(tag-pair)